src/python/pants/backend/jvm/tasks/ivy_imports.py

```diff
@@ -28,6 +29,13 @@ def has_imports(target):
         return isinstance(target, ImportJarsMixin) and target.imported_targets
 
     def execute(self):
+        deprecated_conditional(
+            lambda: True,
+            removal_version="1.30.0.dev0",
+            entity_description="The `imports` goal",
+            hint_message="Contact the Pants team on Slack or pants-devel@googlegroups.com "
+            "if you need this functionality.",
+        )
```

Comment:
It looks like this is a Task rather a Goal, so it's warning for runs on master. Sorry I missed that in the sanity check I did.

Reply:
Sorry, not understanding, what is happening?